tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
 	"compilerOptions": {
-		"target": "ES2015",
+		"target": "ES2021",
```

Comment:
This one concerns me. I'm fairly sure the target was set this way to compile for Node 14 or similar, I'll wait for CI before continuing.